README.md

```diff
@@ -1,260 +1,289 @@
+```
+
+The generated cartogram will be saved in the same format as the map input in the current working directory with the filename:
+
+- `cartogram.json`, if the input map is in GeoJSON format
+- `cartogram.gen`, if the input map is in ArcInfo Generate format
+
+### Examples
+
+For the 2016 US presidential election data, navigate to the `sample_data/` directory.
+
+```
+cd sample_data/
+```
+
+Then run the following command. Note that the sample data has **already been processed** using `cartogram -p`
+```
+cartogram -g usa_contiguous_arcgis.json -a usa_contiguous_electors.csv
+```
+
+To generate an EPS image of the original map and cartogram, use the `-e` option:
+```
+cartogram -eg usa_contiguous_arcgis.json -a usa_contiguous_electors.csv
+```
+Running this command will produce three files:
+
+- `cartogram.json` contains the cartogram in GeoJSON format
+- `map.eps` contains the original map in EPS format
+- `cartogram.eps` contains the generated cartogram in EPS format
+
+On macOS, you can view EPS files with the command:
+```
+open <filename>.eps
+```
+On Linux, run the following instead:
+```
+evince <filename>.eps
+```
+Replace `<filename>` with the name of the file (`map` or `cartogram`) you wish to open.
+
+For our example, `map.eps` should look as follows:
+
+<p align="center">
+<img src ="sample_images/usa_map.png" width="65%">
+</p>
+
+`cartogram.eps` should look like:
+
+<p align="center">
+<img src ="sample_images/usa_cartogram_electors.png" width="65%">
+</p>
+
+### Options
+
+**-e**<br/>
+> Output an EPS file of the original map (`map.eps`) and the cartogram (`cartogram.eps`)
+
+**-d**<br/>
+> Use Gastner-Newman (i.e. diffusion) instead of fast flow-based method
+
+**-i**<br/>
+> Print inverse transform to file
+
+**-g**<br/>
+> Location of file for the original map
+
+**-a**<br/>
+> Location of the area information file. Cannot be used with **-s**.
+
+**-s**<br/>
+> Read the area information from `stdin`, and output the cartogram gen file to `stdout`. Cannot be used with **-a**.
+
+
+## Troubleshooting
+In case you run into problems while building the generator using the automated script, you could try manually carrying out some of the steps.
+
+#### macOS
+1. Install fftw and gcc.
+```
+brew install fftw && brew link fftw
+brew install gcc
+```
+
+2. Note down the version of `gcc` installed. For example, if `gcc-8.1.0` is installed, your version would be `8` (not 8.1.0).
+
+3. Run autogen and configure,  pass it the compiler information
+```
+$ ./autogen.sh
+$ CC=gcc-[your-version-number] ./configure
+```
+
+4. Run make
+```
+$ make
+```
+
+#### Ubuntu/Debian Linux
+1. Make sure everything is up to date. Then install fftw and gcc.
+```
+sudo apt-get update
+sudo apt-get install libfftw3-3 libfftw3-dev
+sudo apt-get install build-essential
+```
+
+2. Make sure you are in the root directory, and then run the remaining steps.
+```
+$ ./autogen.sh && ./configure && make
+```
+
+#### Windows

```

Comment:
If we're including this anyway, it's best to keep it in the dependencies section below the part for Windows 10

Reply:
@krosenfeld I think you missed out the first part of this comment :D 

I'd move this to the dependencies section as it makes more sense there!